src/layouts/ResourceCategory/ResourceCategory.tsx

```diff
@@ -51,22 +61,44 @@ export const ResourceCategory = (): JSX.Element => {
           </Box>
         </Section>
         <Section>
-          <Box w="full">
-            <SectionHeader label="Resource Pages">
-              <CreateButton as={RouterLink} to={`${url}/createPage`}>
-                Create page
-              </CreateButton>
-            </SectionHeader>
-            <SectionCaption icon={BiBulb} label="NOTE: ">
-              Pages are automatically ordered by latest date.
-            </SectionCaption>
-          </Box>
+          {!isPagesEmpty && (
+            <Box w="full">
+              <SectionHeader label="Resource Pages">
+                <CreateButton as={RouterLink} to={`${url}/createPage`}>
+                  Create page
+                </CreateButton>
+              </SectionHeader>
+              <SectionCaption icon={BiBulb} label="NOTE: ">
+                Pages are automatically ordered by latest date.
+              </SectionCaption>
+            </Box>
+          )}
           <Skeleton
             isLoaded={!isLoading}
             w="100%"
             h={isLoading ? "10rem" : "fit-content"}
           >
-            {!pagesData || (!pagesData.length && <Text>No content here</Text>)}
+            {isPagesEmpty && (
+              <VStack spacing={5}>
+                <EmptyBoxImage />
+                <VStack spacing={0}>
+                  <Center textStyle="subhead-1">
+                    There&apos;s nothing here yet.
+                  </Center>
+                  <Center textStyle="body-2">
+                    Create a resource page to get started.
+                  </Center>
```

Comment:
this is kinda weird - the `Center` already guarantees that both are vertically/horizontally centered; we should instead structure this like so:

```ts
<Center>
  <Text textStyle = "subhead-1">
    There&apos;s nothing here yet.
  </Text>
  <Text textStyle = "body-2">
    Create a resource page to get started.
  </Text>
</Center>
```

which prevents ambiguity. wdyt about this change?